phantom_token.c

```diff
@@ -15,6 +15,8 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#define _GNU_SOURCE
```

Comment:
is this needed? We're not using any GNU/Linux extensions in here that I'm aware of.

Reply:
ah, `strcasestr`. I guess we should see about switching to `ngx_strcasestrn` and skip this GNU extra.